protocol.html

```diff
@@ -509,59 +509,62 @@ <h3 property="schema:name"><span>Issue</span>: Solid Notification Vocabulary and
               <section id="discovery" inlist="" rel="schema:hasPart" resource="#discovery">
                 <h3 property="schema:name">Discovery</h3>
                 <div datatype="rdf:HTML" property="schema:description">
-                  <p>The following properties are used for the discovery of a notification channel and its capabilities:</p>
+                  <section id="notification-channel-discovery" inlist="" rel="schema:hasPart" resource="#notification-channel-discovery">
+                    <h4 property="schema:name">Notification Channel Discovery</h4>
+                    <div datatype="rdf:HTML" property="schema:description">
+                      <p about="" id="server-link-resource-notification-channel" rel="spec:requirement" resource="#server-link-resource-notification-channel"><span property="spec:statement">When a server wants to enable applications to discover <a href="#notification-channel">Notification Channels</a> available to a given <a href="https://solidproject.org/TR/protocol#resource">resource</a>, the <span rel="spec:requirementSubject" resource="spec:Server">server</span> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> advertise the associated resources describing the channel information by responding to an HTTP request including a <code>Link</code> header with the <code>rel</code> value of <code>describedby</code> [<cite><a class="bibref" href="#bib-powder-dr">POWDER-DR</a></cite>] and the Description Resource as link target [<cite><a class="bibref" href="#bib-rfc8288">RFC8288</a></cite>].</span></p>
```

Comment:
Powder ([appendix D](https://www.w3.org/TR/2009/REC-powder-dr-20090901/#appD)) puts no constraints on the format of the target of a `describedby` link relation (it could be an image). Should this requirement add a constraint such that a description be at least content-negotiatable as RDF?

Reply:
True. I think this is explained further in notification-channel-resource (RDF Document). Would that suffice?

Reply:
That would suffice for me